lms/resources/_js_config/__init__.py

```diff
@@ -87,6 +87,12 @@ def add_document_url(self, document_url):
                     _query={"document_url": document_url},
                 ),
             }
+        elif document_url.startswith("canvas://"):
```

Comment:
```suggestion
        elif document_url.startswith("canvas://file/"):
```

I think the Canvas file URLs style was chosen so that `canvas://file/*` can be used for Canvas files and `canvas://foo/*` could be used for other things in future. (I think the existing `startswith("blackboard://")` in this same method is also wrong for the same reason: should be `startswith("blackboard://content-resource/")`